File: Proj/C++/Proj2/texasHoldem_Ver4/Player.cpp
/* 
 * File:   Player.cpp
 * Author: Byoung Mo Lee
 * Created on May 27, 2019 10:12 AM
 * Purpose: Player class for Texas Holdem
 * 
 */

#include "Player.h"

int Player::getNumCards() const{
    int scan=0;
    bool found=0;
    
    while(!found && scan<7){
        if(myCards[scan].getFace()==-1) found=1;
        else scan++;
    }
    
    return scan;
}

File: Proj/proj1_texasHoldem/texasHoldem_Ver1.2/Player.php
<?php

/*
 * File:   Player.php
 * Author: Byoung Mo Lee
 * Created on September 24, 2019 01:15 AM
 * Purpose: Texas Holdem
 * 
 */
include ("Card.php");

class Player {

    //private $pName;
    //private $nBalance;
    private $myCards;
    
    

    public function Player() {
        $this->setMyCards();
        //$this->pName="";
        //$this->nBalance=1000;
    }

    public function setMyCards() {
        $this->myCards = array();
    }


    public function getMyCards() {
        return $this->myCards;
    }

    
}
